src/content/docs/en/reference/integrations-reference.mdx

```diff
@@ -527,6 +534,69 @@ export default defineConfig({
 This assumes your integration definition is 1) a `default` export and 2) a function. Ensure this is true before adding the `astro-integration` keyword!
 :::
 
+## `AstroIntegrationLogger`
+
+An instance of the Astro logger, useful to write logs. This logger uses the same [log level](/en/reference/cli-reference/#--verbose)
+configured via CLI.
+
+**Methods available** to write to terminal:
+- `logger.info("Messsage")`;
```

Comment:
```suggestion
- `logger.info("Message")`;
```

Reply:
@matthewp Giving me a taste of my own medicine!  :sweat_smile: 

Drake meme:
Highlighting all these adjacent lines? :no_good: 
Making each line its own suggestion? :+1: